mods/ra/maps/soviet-05/reinforcements_teams.lua

```diff
@@ -0,0 +1,76 @@
+sov1 = {"e2", "e2"}
+sov1Path = { StartPoint.Location, SovietBasePoint.Location }
+
+arty1reinf = { "e3", "e3", "e3", "arty", "arty" }
+if Map.Difficulty == "Easy" then
+	gguy2 = { "jeep", "1tnk", "1tnk" }
+else
+	gguy2 = {"jeep", "jeep", "1tnk", "1tnk", "1tnk" }
+end
+gguy1 = { "e1", "e1", "e1", "e1", "e1" }
+gguyPath = { ReinfNorthLandPoint.Location, RunnerRetreatPoint.Location, NearSovietBasePoint.Location }
+
+AlliedInfantryTypes = { "e1", "e3" }
+AlliedTankTypes = { "jeep", "1tnk" }
+AlliedAttackPath = { GreeceBaseEntryPoint.Location, CrossroadPoint.Location, NWOrefieldPoint.Location, NearSovietBasePoint.Location }
+
+Ships = { }
+GreeceInfAttack = { }
+GGInfAttack = { }
+TankAttackGG = { }
+islanders1 = { "e1", "e1", "e3", "e1", "e3" }
+
+ShipWaypoints = { EastofLittleIsland, WestofLittleIsland, SBNavalAttackPoint }
+InfantryWaypoints = { CrossroadPoint, NWOrefieldPoint, NearSovietBasePoint, SovietBasePoint }
+InfantryGGWaypoints = { PatrolPoint2, BetweenExpBasesPoint, NearSovietExpansionPoint }
+TanksGGWaypoints = { PatrolPoint2, BetweenExpBasesPoint, NearSovietExpansionPoint }
+
+Para = function()
+	local powerproxy = Actor.Create("powerproxy.paratroopers", false, { Owner = player })
+	local units = powerproxy.SendParatroopers(ParaPoint.CenterPosition, false, 32)
+	Utils.Do(units, IdleHunt)
+	powerproxy.Destroy()
+end
+
+Para2 = function()
+	local powerproxy = Actor.Create("powerproxy.paratroopers", false, { Owner = player })
+	local units = powerproxy.SendParatroopers(SovietExpansionPoint.CenterPosition, false, 32)
+	Utils.Do(units, IdleHunt)
+	powerproxy.Destroy()
+end
+
+Gguy1 = function()
```

Comment:
These probably shouldn't be the same names as the tables, with a difference of a capital letter. Can we get better names for these please?